wand/resource.py

```diff
@@ -46,41 +47,36 @@ def terminus():
         library.MagickWandTerminus()
 
 
-#: (:class:`numbers.Integral`) The internal integer value that maintains
-#: the number of referenced objects.
-#:
-#: .. warning::
-#:
-#:    Don't touch this global variable. Use :func:`increment_refcount()` and
-#:    :func:`decrement_refcount()` functions instead.
-#:
-reference_count = 0
+arc = {}
```

Comment:
Does this variable name a meaning elsewhere in the code? It's unclear to me what it means

Reply:
`arc` is short for "automatic reference counter". It's really a reference deallocation map, and should be renamed to reflect that.